File: remove_images.js
const fs = require('fs');
const path = '/src/data/ideas.ts';
let content = fs.readFileSync(path, 'utf8');
content = content.replace(/\s*image:\s*['"].*?['"],?\n/g, '\n');
fs.writeFileSync(path, content);
console.log('Images removed successfully');

File: remove_images.py
import re

with open('/src/data/ideas.ts', 'r') as f:
    content = f.read()

# Remove image property: image: '...', or image: "...", with optional trailing comma
new_content = re.sub(r'\s*image:\s*[\'"].*?[\'"],?\n', '\n', content)

with open('/src/data/ideas.ts', 'w') as f:
    f.write(new_content)
